tests/unit/test_container_to_args_secrets.py

```diff
@@ -302,6 +302,17 @@ async def test_secret_target_matches_secret_name_secret_type_not_env(self):
             "file_secret",
             repo_root() + "/test_dirname/my_secret:/run/secrets/file_secret:ro,rprivate,rbind",
         ),
+        (
+            "relabel",
```

Comment:
This will need 2 cases: for private and shared relabel, because we'll be translating things to what podman understands in this context.